### PR TITLE
Avoid exposing toHtml

### PR DESCRIPTION
**Pull request by iuliaL**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +29,8 @@ You have to import both items (the order matters)
 Add those lines in `index.js`
 
 ```js
-import '@webcomponents/webcomponentsjs/custom-elements-es5-adapter';
```
Comment (jouderianjr) [inline README.md]: I would remove this from the pr and open another pr for those syntax changes.

File: elm.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "name": "PaackEng/elm-google-maps",
     "summary": "Type safe google maps implementation",
     "license": "BSD-3-Clause",
-    "version": "1.0.0",
```
Comment (jouderianjr) [inline elm.json]: Don't bump the version here, we should do it in another pr/commit as well

File: examples/src/index.js [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
-import * as serviceWorker from './serviceWorker';
-import '@webcomponents/webcomponentsjs/custom-elements-es5-adapter';
-import '@google-web-components/google-map';
+import "./main.css";
+import { Elm } from "./Main.elm";
+import * as serviceWorker from "./serviceWorker";
+import "@webcomponents/webcomponentsjs/custom-elements-es5-adapter";
+import "@google-web-components/google-map";
 
-const flags =
```
Comment (jouderianjr) [inline examples/src/index.js]: I would remove this from the pr and open another pr for those syntax changes.

File: src/GoogleMaps/Polygon.elm [hunk 4]
```diff
@@ -117,31 +111,31 @@ E.g `"rbg(255,255,0)", "orange" or "#ff00ff"`
...
-    Polygon <| IPolygon.onClick msg polygon
+onClick msg polygon =
+    IPolygon.onClick msg polygon
 
 
 {-| You should not use this function, GoogleMaps.Map uses it in order to render the polygon.
 -}
 toHtml : Polygon msg -> Html msg
-toHtml (Polygon polygon) =
+toHtml polygon =
```
Comment (jouderianjr) [inline src/GoogleMaps/Polygon.elm]: we can remove this function right?

Comment (iuliaL) [reply src/GoogleMaps/Polygon.elm]: Yes. Leftover

File: elm.json (hunk 2 above)
Comment (jouderianjr) [reply elm.json]: can you revert this change? I mean, the whole file change. this is a generated file, then if we add another dependency, the file will probably add the 4 spaces tab again :'(

File: README.md (hunk 1 above)
Comment (jouderianjr) [reply README.md]: It's missing this one
